mars/oscar/backends/pool.py

```diff
@@ -1350,6 +1350,12 @@ async def monitor_sub_pools(self):
                             event.set()
                     except asyncio.CancelledError:
                         raise
+                    except RuntimeError as ex:  # pragma: no cover
+                        if "shutdown" not in str(ex):
```

Comment:
Just identify `shutdown` seems not so accurate, maybe we should add all the error msg.